### ISSUE TITLE
Automatically create user when anonymously user comments

### ISSUE THREAD

**Issue by NJavokhir** [globe-and-citizen/Celebrity-Fanalyzer#70]
We have to create anonymous user automatically in our database when anonym user comments. We set _browserID_ for every anonym user.

**Comment by arnonrdp** [comment]
I recently created a function (`fetchUserIp`) and I believe we can use that instead of the _browserId_.

**Comment by stravid87** [comment]
@hermannleboss, you face the same issue when collecting likes / dislikes -- what is our general strategy for recording anonymous users in the DB?

**Comment by NJavokhir** [comment]
> I recently created a function (`fetchUserIp`) and I believe we can use that instead of the _browserId_.

Will do.

**Comment by arnonrdp** [comment]
> @hermannleboss, you face the same issue when collecting likes / dislikes -- what is our general strategy for recording anonymous users in the DB?

I updated the likes/dislikes functions and we are now storing the user IP instead of browser ID.

But I am going to update it again to use user IP only if the user is not authenticated. Else we will use the UID Firebase Auth gives to us. Because it is unchangeable. But the IP can be changed.

**Comment by hermannleboss** [comment]
> @hermannleboss, you face the same issue when collecting likes / dislikes -- what is our general strategy for recording anonymous users in the DB?

I created a browserId which can be useful, but when I discussed it with @arnonrdp he said we can build something more powerful.

**Comment by stravid87** [comment]
Yes, I discussed with Arnon. He has a simple solution to get the IP.
The IP we can do more with than the browser ID. But, I countered that I think we should use a hash of the IP -- that way we get the advantage of a non-specific / Random BrowserID but ALSO the robustness of the IP address

**Comment by stravid87** [comment]
So hash(IP) = anonymousID
This way we can worry less about privacy.

**Comment by hermannleboss** [comment]
OK it's good

**Comment by stravid87** [comment]
> This issue is no longer needed as we are now able to comment with anonymous users without the need to create a new user Firestore Document.

Javokhir and I looked at this today.
